tests/test_ain_utils.py

```diff
@@ -0,0 +1,196 @@
+from unittest import TestCase
+from ain.utils import *
+from .data import (
+    address,
+    pk,
+    sk,
+    mnemonic,
+    mnemonicPrivateKey,
+    mnemonicPublicKey,
+    mnemonicAddress,
+    checksumAddresses,
+    message,
+    correctSignature,
+    tx,
+    txDifferent,
+)
+
+class TestKeccak(TestCase):
+    def testKeccak(self):
+        msg = "0x3c9229289a6125f7fdf1885a77bb12c37a8d3b4962d936f7e3084dece32a3ca1"
+        r = "82ff40c0a986c6a5cfad4ddf4c3aa6996f1a7837f9c398e17e5de5cbd5a12b28"
+        hash = keccak(msg)
+        self.assertEqual(hash.hex(), r)
+
+    def testKeccakWithoutHexprefix(self):
+        msg = "3c9229289a6125f7fdf1885a77bb12c37a8d3b4962d936f7e3084dece32a3ca1"
+        r = "22ae1937ff93ec72c4d46ff3e854661e3363440acd6f6e4adf8f1a8978382251"
+        hash = keccak(msg)
+        self.assertEqual(hash.hex(), r)
+
+class TestByteToHex(TestCase):
+    def testByteToHex(self):
+        byt = bytes.fromhex("5b9ac8")
+        hex = bytesToHex(byt)
+        self.assertEqual(hex, "0x5b9ac8")
+
+    def testByteToHexEmptyBytes(self):
+        byt = bytes(0)
+        hex = bytesToHex(byt)
+        self.assertEqual(hex, "0x")
+
+class TestIsValidPrivate(TestCase):
+    SECP256K1_N: int = 0xfffffffffffffffffffffffffffffffebaaedce6af48a03bbfd25e8cd0364141
+
+    def testIsValidPrivateShortInput(self):
+        tmp = "0011223344"
+        self.assertFalse(isValidPrivate(bytes.fromhex(tmp)))
+
+    def testIsValidPrivateTooBigInput(self):
+        tmp = "3a443d8381a6798a70c6ff9304bdc8cb0163c23211d11628fae52ef9e0dca11a001cf066d56a8156fc201cd5df8a36ef694eecd258903fca7086c1fae7441e1d"
+        self.assertFalse(isValidPrivate(bytes.fromhex(tmp)))
+    
+    def testIsValidPrivateInvalidCurveZero(self):
+        tmp = "0000000000000000000000000000000000000000000000000000000000000000"
+        self.assertFalse(isValidPrivate(bytes.fromhex(tmp)))
+
+    def testIsValidPrivateInvalidCurveEqN(self):
+        self.assertFalse(isValidPrivate(self.SECP256K1_N.to_bytes(32, "big")))
+    
+    def testIsValidPrivateInvalidCurveGtN(self):
+        self.assertFalse(isValidPrivate((self.SECP256K1_N + 1).to_bytes(32, "big")))
+
+    def testIsValidPrivateWork(self):
+        self.assertTrue(isValidPrivate((self.SECP256K1_N - 1).to_bytes(32, "big")))
+
+# TODO(kriii): Add tests after implement `isValidPrivate`
```

Comment:
minor: Could you add period (.) at the end of not only this one but all the TODO instances?
https://www.notion.so/comcom/ComCom-Developer-s-Guide-ad57ebc31ef94040a4f6e71d9c2042c5

Reply:
I added all periods. Sorry!